src/sentry/tasks/performance_detection.py

```diff
@@ -55,6 +56,10 @@ def get_detection_settings():
             "duration_threshold": 500.0,  # ms
             "allowed_span_ops": ["db", "http"],
         },
+        DetectorType.UNCOMPRESSED_ASSET_SPAN: {
+            "size_threshold_bytes": 500 * 1024,
+            "allowed_span_ops": ["resource.css", "resource.script"],
```

Comment:
I think it's possible we'll also want some timing threshold on top of this in practice since 500kb might be fine depending on your use case or user conditions.